inventory/group_vars/all.yml

```diff
@@ -15,21 +15,27 @@ ssh_pubkey: "{{ lookup('file', '~/.ssh/{{ ssh_pubkey_name }}') }}"
 protocol: https
 
 git_repo: https://github.com/openfoodfoundation/openfoodnetwork.git
-git_version:  v1.7.0
+git_version:  v1.8.6
```

Comment:
Do we want to keep updating this? Or do we want to use a "latest" branch with the last release? On the other hand, it's good to document here the last tested version. People may need to update stuff if they want a newer OFN version. And you can always update the OFN after the initial provisioning. So I guess it's good to keep the version number here.